Nos.xcodeproj/project.pbxproj

```diff
@@ -3232,8 +3232,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/SDWebImage/SDWebImageSwiftUI";
 			requirement = {
-				kind = upToNextMajorVersion;
-				minimumVersion = 2.0.0;
+				branch = bugfix/webimage_isAnimating_binding;
```

Comment:
We'll definitely want to update this before merging the PR. But I thought there's a chance they might get the [SDWebImageUI PR](https://github.com/SDWebImage/SDWebImageSwiftUI/pull/333) merged and released before _this_ PR is reviewed and ready to merge. Either way, I'll set this to a released version of SDWebImageSwiftUI before merging -- the only question is whether I'll have to use a workaround in our code or not.

Reply:
Updated -- we're on release 3.1.2 now!